frontend/src/features/permits/hooks/hooks.ts

```diff
@@ -382,16 +382,20 @@ export const useApplicationsInProgressQuery = ({
   page = 0,
   take = 10,
   searchString = "",
-  sorting = [],
+  orderBy = [],
+}: {
+  page?: number,
```

Comment:
Minor fix: change the ',' to ';' for type definitions:
`{
    page?: number;
    take?: number;
    searchString?: string;
    orderBy?: SortingConfig[];
}`

Reply:
updated